contrib/test_jenkins.sh

```diff
@@ -239,6 +239,36 @@ build_debug() {
 }
 
 #
+# Build CUDA
+#
+
+build_cuda() {
+    echo 1..1 > build_cuda.tap
+    if module_load dev/cuda
+    then
+        if module_load dev/gdrcopy
+        then
+            echo "==== Build with enable cuda ===="
+            ../contrib/configure-devel --prefix=$ucx_inst --with-cuda --with-gdrcopy
```

Comment:
do we need to test that cuda was actually built?

Reply:
configure fails if not able to build cuda, do we need more than this?